behave/behave.ini

```
@@ -5,4 +5,4 @@ stdout_capture = False
 
 
 [behave.userdata]
-BASE_URL = 'https://demoqa.com/books'
\ No newline at end of file
+BASE_URL = https://demoqa.com/books
```

```suggestion
BASE_URL = https://demoqa.com/
```
Pra ficar igual ao Robot, a base url deveria abrir só a demoQA e daí ter um step que acessaria a bookstore pelo card